mem/module_info.c

```diff
@@ -59,4 +60,61 @@ inline void update_module_stats(long mem_used, long real_used, int frags, int gr
 	update_stat(memory_mods_stats[group_idx - 1].memory_used, mem_used);
 	update_stat(memory_mods_stats[group_idx - 1].real_used, real_used);
 #endif
+}
+
+int alloc_group_stat(void) {
+	int size_prealoc, j, one_full_entry, groups;
+	char *start;
+	struct module_info* new_stats_vec;
+#ifndef SHM_SHOW_DEFAULT_GROUP
+	groups = mem_free_idx - 1;
+#else
+	groups = mem_free_idx;
+#endif
+
+	one_full_entry = 3 * (sizeof(stat_var) + sizeof(stat_val));
+	size_prealoc = groups * sizeof(struct module_info) + groups * one_full_entry;
+
+#ifndef DBG_QM_MALLOC
```

Comment:
I think this flag is obsolete. Please check if this flag is still present in the latest version.